recipes/r-flow/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = '0.1.0.9000' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-flow
+  version: {{ version|replace("-", "_") }}
+
+source:
+
+  git_url: https://github.com/moodymudskipper/flow
+  git_tag: v0.1.0
```

Comment:
see https://github.com/conda-forge/staged-recipes/pull/21266/files#r1032930508

Reply:
Thanks let me try this